config_hub/finetune/falcon-7b/lora.yaml

```diff
@@ -114,6 +114,9 @@ eval:
   # Number of iterations (type: int, default: 100)
   max_iters: 100
 
+  # Whether to evaluate on the validation set at the beginning of the training
+  initial_validation: false
```

Comment:
Is it really needed to add an argument for this? Why not just use the initial sanity check as the initial validation loss?